agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,6 +123,50 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_dependencies=$(getconf.py -l stockpile_dependencies packages)
+	stockpile_log=$(getconf.py stockpile_log stockpile)
+
+	# install stockpile and its dependencies
+	for package in $stockpile_dependencies; do
+       		which $package &>/dev/null
+        	if [[ $? != 0 ]]; then
+                	debug_log "[$script_name] $package is not installed on the system, exiting"
+                	echo "$package is not installed on the system, please install it to collect stockpile data"
+         		exit 1
+        	fi
+	done
+	if [[ -z "$stockpile_path" ]]; then
+		stockpile_path=$(getconf.py pbench_install_dir DEFAULT)
+	fi
+	if [[ -d "$stockpile_path/stockpile" ]]; then
+        	rm -rf $stockpile_path/stockpile
+	fi
+
+	# set stockpile log file and output paths
+	if [[ -z "$stockpile_log" ]]; then
+                stockpile_log=$dir/stockpile.log
+        fi
+	if [[ -z "$stockpile_output_path" ]]; then
+        	stockpile_output_path=$dir/stockpile.json
+	fi
+
+	# generate inventory and run stockpile
+	generate_inventory stockpile > $INVENTORY
+	debug_log "[$script_name]Collecting stockpile data"
+	ansible-playbook -vv --extra-vars "stockpile_user=$stockpile_user local_remote_user=$stockpile_local_remote_user host_remote_user=$stockpile_host_remote_user stockpile_output_path=$stockpile_output_path" -i $INVENTORY $stockpile_path/stockpile/stockpile.yml 2>&1 > "$stockpile_log"
```

Comment:
Can you add a `debug_log` line to capture the `ansible-playbook` command line invocation?  Perhaps write the full command to a file, and then invoke the file?  Like we do in benchmark scripts?